pyigm/surveys/lls_literature.py

```diff
@@ -0,0 +1,1078 @@
+"""
```

Comment:
Yes, it is.  I think I updated it as we went to clean those
issues.  Can you remember any specific one so that
I can double check?